### PR TITLE
Add accounting fields to TARs

### LINKED ISSUE #2295: Add accounting fields to TAR

### PR DESCRIPTION
**Pull request by kberzinch**
Closes #2295

### CODE REVIEW

File: app/Nova/Travel.php [hunk 1]
```diff
@@ -208,6 +208,22 @@ public function fields(Request $request): array
                             ' If this is not applicable, enter 0.'
                         )
                         ->hideFromIndex(),
+
+                    Text::make('Workday Project Number', 'tar_project_number')
```
Comment (JoshOldenburg) [inline app/Nova/Travel.php:212]: Would this make more sense as a dropdown?

**Review by JoshOldenburg — COMMENTED** [review]
Looks good to me. One quick question

File: app/Nova/Travel.php (hunk 1 above)
Comment (kberzinch) [reply app/Nova/Travel.php:212]: I feel like a dropdown would somewhat discourage getting the real answer from GT.

Comment (JoshOldenburg) [reply app/Nova/Travel.php:212]: Makes sense.

**Review by JoshOldenburg — APPROVED** [review]

**Pull request merged**